src/pages/EmployeeCreation/components/UploadedFile.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
```

Comment:
<img width="923" alt="image" src="https://github.com/SE2-WS23/Skillverse-Web/assets/65715000/a79bc05d-15a5-4556-9884-4240e28e0e61">

If you upload a file, the following error occurs